executors/Python.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+exec python "$1"
```

Comment:
Looks like python buffers `stdout`/`stderr` output by default. Running it with the `-u` flag disables this.
```suggestion
exec python -u "$1"
```
I have tried this and can see the live output!

Reply:
Awesome! Thanks for trying it out